multitenancy/tenant-manager-datamodel/src/main/java/io/apicurio/multitenant/api/beans/TenantStatusValue.java

```diff
@@ -0,0 +1,47 @@
+
+package io.apicurio.multitenant.api.beans;
+
+import java.util.HashMap;
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum TenantStatusValue {
+
+    READY("READY"),
+    TO_BE_DELETED("TO_BE_DELETED"),
+    DELETED("DELETED");
```

Comment:
Maybe add a documentation comment, even if it's obvious.

Reply:
this is generated from Studio, and there is already doc about this, but not in the enum class.